l1-contracts/src/core/libraries/DataStructures.sol

```diff
@@ -79,4 +81,20 @@ library DataStructures {
     bool ignoreDA;
     bool ignoreSignatures;
   }
+
+  struct EpochProofQuote {
```

Comment:
For the `ecrecover` to be safe the message it is signing over should include the address that we expect to recover. Otherwise valid signature for a different message can also recover an address, just a different one. e.g., include the `bondProvider` in the quote.